src/utils/re_frame.cljs

```diff
@@ -76,6 +77,14 @@
     (swap! handler-nesting-level dec)
     res))
 
+(defn delay-render
+  [content]
+  (let [render? (reagent/atom false)]
+    (js/setTimeout #(reset! render? true) 0)
```

Comment:
The RN doc says `setTimeout` or `setImmediate` may delay animations. `runAfterInteractions` doesn't suffer from that, although only experimentation can tell if it works better.

Source: https://reactnative.dev/docs/next/timers

Reply:
we don't have animations here, what kind of animation they might delay? setTimeout just put code in the next js task

Reply:
`delay-render` is a general component, and `content` can be anything. How can this function know there's no animation?

Reply:
that's expected, if you delay component :)

Reply:
but i think documentation is old, and that about times there were only js animation based on timers etc , now with reanimated it's not actual anymore

Reply:
You said there are no animations here, but there's no way to know from here. My point is just that `setTimeout` with 0 is a workaround that may delay animations as pointed by the RN docs, and there's no way for this function to know what `content` does.

At the very least I think it should be investigated how this solution fares against `content` that has animations versus using `runAfterInteractions`.

This of course in a future branch/PR because this one is already merged.

Reply:
i said that because i believe docs says that if you have animation in your code somewhere , like in the parent component it will be affected by setTimeout because Timeout has a priority, but in our case if you use delay you know that entire component will be delayed, so offcourse animation will be delayed as well, if content has it, so it should be obvious for developer who uses delay, but again, i don't say it's a general solution, so probably we should move it in separate namespace and add description how to use it, for now it must be used only in this current case, and in other cases it should be used with understanding how it works

Reply:
> and add description how to use it, for now it must be used only in this current case,

That would be great @flexsurfer.

> and in other cases it should be used with understanding how it works

👍🏼 perfect